mitodl/open-discussions
# 404 for removed comments

#### Pre-Flight checklist

#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/issues/3087

#### What's this PR do?
This adds a server-side 404 page for removed comments for non-moderator users

#### How should this be manually tested?
Make a comment as a non-moderator user
Get the share url for the comment it should be something like `http://localhost:8063/c/public_channel/5/i-like-robotics/comment/3/`
Mark the comment as spam using 
 `docker-compose run web ./manage.py reclassify_spam --spam --comment-id <object id of comment>`
Go to the comment url. You should be able to see the comment as a moderator but get a 404 as a non-mderator or anonymous user
